### ISSUE TITLE
profile_list → wrapspawner or jupyterhub

### ISSUE THREAD

**Issue by rkdarst** [jupyterhub/zero-to-jupyterhub-k8s#1821]
Opening this mainly to track the related ideas so they don't get forgotten.

### Proposed change

Transferring from https://github.com/jupyterhub/kubespawner/pull/393#issuecomment-703317048 could be relevant, discusses wrapspawner and how the profile list could be moved to wrapspawner.  The profile_list has a lot of overlap with [wrapspawner](https://github.com/jupyterhub/wrapspawner), info in the linked issue.

Another option is to move all of this to core JupyterHub: https://github.com/jupyterhub/jupyterhub/issues/2724

### Alternative options

do nothing (and/or not fill this section in issue template)

### Who would use this feature?

Other spawners who need essentially the same feature, and there are already two implementations

### (Optional): Suggest a solution

"obvious", but needs discussion and work.  I don't expect to have much time myself, though...

**Comment by consideRatio** [comment]
@rkdarst I'm on an issue triaging spree in this repo, and would say that this is a question for KubeSpawner, not for the repo that consumes KubeSpawner, right?

**Comment by rkdarst** [comment]
Yes, oops... I didn't see the repo I was opening on.  Please triage.

>
